test/e2e/tests/test_ca_activation.py

```diff
@@ -600,6 +600,8 @@ def test_subordinate_ca_activation(self, acmpca_client, subordinate_ca_activatio
         assert base64.b64decode(api_response[complete_certificate_chain_secret.key]).decode("ascii") == complete_certificate_chain
 
     def test_disabled_activation(self, acmpca_client, simple_ca_activation_status_disabled):
+        time.sleep(10)
```

Comment:
The sleep shouldn't happen here, more at the `simple_ca_activation_status_disabled` level